packages/components/psammead-navigation/src/index.stories.jsx

```diff
@@ -163,11 +176,36 @@ const navigationStory = (
   );
 };
 
+const dropdownStory = () => ({ dir, script, service }) => (
+  <Dropdown>
+    <DropdownUl>
+      {pidginNavData.map((item, index) => {
+        const active = index === 3;
+        const { title, url } = item;
+        return (
+          <DropdownLi
+            script={script}
+            service={service}
+            key={title}
+            dir={dir}
+            url={url}
+            active={active}
+            currentPageText="Current page"
+          >
+            {title}
+          </DropdownLi>
+        );
+      })}
+    </DropdownUl>
+  </Dropdown>
+);
+
 const storiesWithoutBrand = storiesOf(
   'Components|Navigation/without brand',
   module,
 )
   .addDecorator(withKnobs)
+  .addDecorator(ampDecorator)
```

Comment:
Everything looks really good to me 👍 . Just wondering if adding an ampDecorator to all the storiesWithoutBrand was on purpose?

Reply:
Yes, this was on purpose in order to make the AMP menu button work. If you click on the AMP button now, the menu icon will change